packages/core/src/codewhisperer/service/transformByQ/humanInTheLoopManager.ts

```diff
@@ -0,0 +1,102 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import * as vscode from 'vscode'
+import * as os from 'os'
+import path from 'path'
+import { FolderInfo } from '../../models/model'
+import { fsCommon } from '../../../srcShared/fs'
+import { createPomCopy, replacePomVersion } from './transformFileHandler'
+import { IManifestFile } from '../../../amazonqFeatureDev/models'
+import { getLogger } from '../../../shared/logger'
+
+export class HumanInTheLoopManager {
```

Comment:
what does this class do? needs a docstring

Reply:
Helps encapsulate all the interactions for artifact management during this feature. Makes it easier to cleanup